src/Header/SetCookie.php

```diff
@@ -264,8 +264,6 @@ public function __construct(
         $version = null,
         $sameSite = null
     ) {
-        $this->type = 'Cookie';
```

Comment:
missing property should be added instead, make public

Reply:
Would also mark it `@deprecated`

Reply:
@Ocramius @samsonasik I added the property for SetCookie class, and it's marked as deprecated. Should I make it for Response's headerSent property too?

Reply:
Yes please: the goal is to not accidentally break BC here :)

Reply:
@Ocramius Done